litestar/middleware/compression/middleware.py

```diff
@@ -156,7 +84,7 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
     def create_compression_send_wrapper(
         self,
         send: Send,
-        compression_encoding: Literal[CompressionEncoding.BROTLI, CompressionEncoding.GZIP],
+        compression_encoding: Literal[CompressionEncoding.BROTLI, CompressionEncoding.GZIP] | str,
```

Comment:
This type hint is changed everywhere since `encoding` on `CompressionFacade` is kept as a string.